obspy/signal/spectral_estimation.py

```diff
@@ -1849,6 +1940,36 @@ def plot(self, filename=None, show_coverage=True, show_histogram=True,
                     xdata = periods
                 ax.plot(xdata, noise_model, '0.4', linewidth=2, zorder=10)
 
+        if show_earthquakes is not None:
+            min_mag, max_mag, min_dist, max_dist = show_earthquakes
+            for key, data in earthquake_models.items():
+                magnitude, distance = key
+                frequencies, accelerations = data
+                accelerations = np.array(accelerations)
+                frequencies = np.array(frequencies)
+                periods = 1.0 / frequencies
+                # Eq.1 Cauzzi & Clinton (2013)
+                ydata = accelerations / (periods ** (-.5))
+                ydata = 20 * np.log10(ydata / 2)
```

Comment:
This conversion from m/s/s to dB has been checked by @ccauzzi and @jclinton.

Reply:
Okay, yes.  You are converting spectra to densities.  Usually, folks go the other way, but this works.  I think that was my confusion.  

Would you be willing to put a comment or a note?  I apologize for making this such an issue, but it has been done incorrectly in various papers, which has produced confusion.

These curves will be very useful!

Reply:
@aringler-usgs : thanks a lot, but could you suggest the kind of comment that should be added? And where exactly?

Reply:
Maybe just change your current comment to the following?

```
# Eq.1 from Clinton and Cauzzi (2013) converts power to density
```

That would be sufficient for me.  This helps someone know there is a unit change here.

Reply:
OK thanks a lot.